src/axom/spin/tests/spin_bvh.cpp

```diff
@@ -836,11 +1470,10 @@ AXOM_CUDA_TEST( spin_bvh, single_box3d_cuda )
   constexpr int BLOCK_SIZE = 256;
   using exec  = axom::CUDA_EXEC< BLOCK_SIZE >;
 
-  check_single_box3d< exec, float >( );
   check_single_box3d< exec, double >( );
```

Comment:
Does the order of ``float`` and ``double`` matter?

Reply:
No, the order doesn't matter.